### ISSUE TITLE
md-switch is broken

### ISSUE THREAD

**Issue by hrushipatil1209** [angular/material#816]
Hi, 

It seems the md-swittch directive is broken with the new release.

When I add 

``` html
<md-switch ng-model="machine.state" 
           aria-label="Power" 
           ng-true-value="'ON'" 
           ng-false-value="'OFF'" >
                {{ machine.state }}
</md-switch>
```

The switch does not work.
I get the following warning in the console
ARIA: Attribute " aria-label ", required for accessibility, is missing on node: <div class=​"md-switch-thumb md-default-theme" role=​"checkbox">​…​</div>​
